plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/struct/DBStructUtils.java

```diff
@@ -351,41 +353,53 @@ public static String mapTargetDataType(DBSObject objectContainer, DBSTypedObject
                             }
                         }
                         if (targetType == null) {
-                            if (typeNameLower.contains("float") ||
-                                typeNameLower.contains("real") ||
+                            String intDataType = "int";
+                            String integerDataType = "integer";
+                            String floatDataType = "float";
+                            String realDataType = "real";
```

Comment:
These are not being reassigned anywhere. Please make them `static final` constants.

Reply:
It's about code smell. No one does that, everyone use class constants.

Reply:
I will repeat it - Util class is not an appropriate place for the class constants. It is a commonplace. It has many not bounded methods.